app/lib/test_applications.rb

```diff
@@ -221,10 +221,11 @@ def create_application_to_courses(
         SubmitApplication.new(@application_form).call
 
         @application_form.application_choices.each do |choice|
-          choice.update_columns(
+          choice.update(
             sent_to_provider_at: time,
-            reject_by_default_at: time + rand(3..30).days,
+            reject_by_default_at: time + rand(10..40).days,
```

Comment:
Is there a reason we are keeping the 10-40 days random part? This test generation is used for stuff other than sandbox, but I can't imagine why it would be useful to generate RBDs that aren't just (submission date + 40 days).end_of_day

Reply:
Yea I wasn't sure about this either, I would prefer we keep to the default to avoid confusion. My thoughts were around testing in other environments and wanting to stagger those but tbh might be best to drop this. I'll change

Reply:
They should be staggered anyway because (hopefully) we stagger the submission dates. Sounds good 👍